### DISCUSSION TITLE
Store credentials in database

### DISCUSSION THREAD

**Discussion by devrandomzero** [netflie/laravel-notification-whatsapp#16] [General]
Hi, I've got a multitenant setup and I would like to get the `WHATSAPP_FROM_PHONE_NUMBER_ID` and `WHATSAPP_TOKEN` values from the app database (they differ for each tenant), is it possible?

Thank you for the help

**Comment by hybrid1969** [comment]
Any movement on this as I am in the same position?

**Comment by alvaro-canepa** [comment]
Hello, any news with this?
